src/main/resources/swagger.api/schemas/locationMappingForAllLibrariesDTO.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Location mapping",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Mapping id",
+      "type": "string",
+      "format": "uuid"
+    },
+    "innReachLocationId": {
+      "description": "Inn-Reach location id",
+      "type": "string",
+      "format": "uuid"
+    },
+    "locationId": {
+      "description": "Location id",
+      "type": "string",
+      "format": "uuid"
+    },
+    "libraryId": {
+      "description": "Library id",
+      "type": "string",
+      "format": "uuid"
+    },
+    "metadata": {
+      "description": "Entity metadata",
+      "type": "object",
+      "$ref": "metadata.json"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "innReachLocationId",
+    "locationId",
+    "libraryId"
+  ]
+}
```

Comment:
is there actually a need for a new DTO? This one has the same structure as existing locationMappingDTO.json with just one additional field `libraryId`. I suggest to simply update existing DTO with this field, just verify that it doesn't brake existing CRUD for location mappings

Reply:
yep, it works